src/main/java/org/team1540/robot2019/datastructures/twod/Twist2D.java

```diff
@@ -0,0 +1,46 @@
+package org.team1540.robot2019.datastructures.twod;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+/**
+ * 2D twist data structure class
+ */
+public class Twist2D {
+
+    public static final Twist2D ZERO = new Twist2D(0, 0, 0);
+
+    private final double x;
+    private final double y;
+    private final double omega;
+
+    /**
+     * @param x Linear velocity in X direction in meters per second
+     * @param y Linear velocity in Y direction in meters per second
+     * @param omega Angular velocity counter-clockwise in radians per second
+     */
+    public Twist2D(double x, double y, double omega) {
+        this.x = x;
+        this.y = y;
+        this.omega = omega;
+    }
+
+    public double getX() {
+        return x;
+    }
+
+    public double getY() {
+        return y;
+    }
+
+    public double getOmega() {
+        return omega;
+    }
+
+    public void putToNetworkTable(String networkTablesPath) {
```

Comment:
Maybe provide a `NetworkTable` here instead of a path, so we aren't getting the instance all the time?

Reply:
What's wrong with getting the instance? It's literally just returning the same pointer every time.

Reply:
That's true.